README_dev.md

```diff
@@ -2,6 +2,7 @@
 
 1. Download and install [Node JS](https://nodejs.org/en/download/)
 2. run `npm i`
+3. run `npm build:watch`
```

Comment:
missing `run`

Also, do I actually need to call this. For me another shell opened automatically.